ghost/core/test/e2e-webhooks/posts.test.js

```diff
@@ -154,4 +154,57 @@ describe('post.* events', function () {
                 }
             });
     });
+
+    it('post.scheduled event is triggered', async function () {
+        const webhookURL = 'https://test-webhook-receiver.com/post-scheduled/';
+        await webhookMockReceiver.mock(webhookURL);
+        await fixtureManager.insertWebhook({
+            event: 'post.scheduled',
+            url: webhookURL
+        });
+
+        const res = await adminAPIAgent
+            .post('posts/')
+            .body({
+                posts: [{
+                    title: 'Testing post.scheduled webhook',
+                    status: 'draft'
+                }]
+            })
+            .expectStatus(201);
+
+        const id = res.body.posts[0].id;
+        const scheduledPost = res.body.posts[0];
+        scheduledPost.status = 'scheduled';
+        const publishedTime = new Date();
+        publishedTime.setDate(publishedTime.getDate() + 1);
+        scheduledPost.published_at = publishedTime.toISOString();
```

Comment:
For date calculations in the codebase there's a preference to use a date library - [like here](https://github.com/TryGhost/Ghost/blob/34439f2e28e4603e9ef30c86954099d56a7c9ef2/ghost/core/test/regression/api/admin/schedules.test.js#L36).

The following: 
```js
moment().add(30, 'seconds').toDate()
```

is easier to read and later migrate to a different library (we have been trying to [migrate away to Luxon from momentjs](https://github.com/TryGhost/Ghost/issues/13648)).

Reply:
Thanks for the suggestion! I updated the date calculation using the date library.